avocado/core/runner.py

```diff
@@ -428,38 +428,43 @@ def run_suite(self, test_suite, mux, timeout=0, replay_map=None,
 
         index = -1
         for test_template in test_suite:
```

Comment:
Well I have just one question, what happens, when the user hits `ctrl+c` when the `for` loop assigns the new `test_template`? :wink: